### ISSUE TITLE
Markdown tables cells not rendered

### ISSUE THREAD

**Issue by keiranmraine** [byrnereese/mkdocs-bootstrap4#4]
Tables don't render with cell borders:

e.g.
```
|      | Original only  | shared       | New only       |
|------|----------------|--------------|----------------|
| Pass | 0 | 2 | 0 |
| All  | 0  | 6107  | 0 |
```

Should appear as:

|      | Original only  | shared       | New only       |
|------|----------------|--------------|----------------|
| Pass | 0 | 2 | 0 |
| All  | 0  | 6107  | 0 |

(not interested in alternating row highlight)

**Comment by byrnereese** [comment]
This is by design. If you would like to style your tables differently, I recommend creating a custom stylesheet. 

See also:
https://www.mkdocs.org/user-guide/configuration/#extra_css

**Comment by aparcar** [comment]
Isn't bootstrap4 styling tables?

**Comment by byrnereese** [comment]
I did nothing to customize the display of tables, preferring to adopt whatever the default styles might have been. But yes, Bootstrap 4 is styling the tables. By default markdown doesn't create markup that conforms to the bootstrap table standard. So I built a plugin that converts naked markdown tables into bootstrap tables. That plugin is: https://github.com/byrnereese/mkdocs-bootstrap-tables-plugin

The plugin simply adds a <th> row, and adds appropriate class names to each column and to the table itself.
